content/backend/typescript-apollo/9-deployment.md

```diff
@@ -0,0 +1,692 @@
+      id
+      name
+      email
+    }
+  }
+}
+
+mutation SecondPostMutation {
+  post(url: "www.prisma.io", description: "Next-generation Node.js and TypeScript ORM") {
+    id
+    description
+    url
+    postedBy {
+      id
+      name
+      email
+    }
+  }
+}
+
+query FeedQuery{
+  feed {
+    count
+    links {
+      id
+      createdAt
+      description
+    }
+  }
+}
+
+
+
+```
+
+</Instruction>
+
+![Running API operations in production server](https://i.imgur.com/hNQsB3l.gif) 
+
+
+### Exploring your data in Prisma Data Platform 
+
+Previously, you were using Prisma Studio to directly interact with your data. However, Prisma Studio is meant to be used with your development or test database and is not a great option to interact with production data. To solve this problem, Prisma has a hosted version of Prisma Studio inside the [Prisma Data Platform](https://cloud.prisma.io/), called the **Data Browser**.  
+
+To get started, go to https://cloud.prisma.io/ and click **Continue with GitHub**. The Prisma app will then ask for permission to read your Email address. Click on **Authorize Prisma** to proceed. In the dashboard, click on **New Project**. Then you will need to click on **Add an Organization or Account** under the dropdown in **GitHub Account**. Since you will be importing an existing GitHub project to Prisma, you will need to install Prisma and give it read and write access to that GitHub repository. 
+
+In case you are a part of multiple organizations, you will first be asked to choose which account you want to use. Then you will have to choose if you want to install Prisma on *all repositories* or *select repositories*. Choose **Only select repositories** and choose the **hackernews-typescript** repository. Then click on **Install**. 
+
+Now you will be redirected back to the **Configure project** page. Choose **Import a Prisma repository** and then select the **hackernews-typescript** repository. Then give a suitable name to your project (this is the name of the Prisma Data Platform project, not to be confused with the name of the repository). Now click **Next** to continue. 
+
+Now you will be shown the **Configure environment** page. Since you already have a database created, choose **Use my own database**. Then paste the connection string you received from Heroku. You can leave the other options to their default value. Finally, click on **Create project** to proceed. You will be given a connection URL to the [Prisma Data Proxy](https://www.prisma.io/docs/concepts/data-platform/data-proxy), which you can **skip** as you won't be using the Data Proxy at the moment. 
+
+Alright! With all the configuration out of the way, click on the **Data Browser** on the left-hand side to get a hosted version of the Prisma Studio. The next time you log in to the Prisma Data Platform, this project should be available for you to access. It will also be synced with any schema changes pushed to the `master` branch on GitHub. 
```

Comment:
Alright! With all the configuration out of the way, click on the **Data Browser** buton on the left-hand side to get a hosted version of the Prisma Studio. The next time you log in to the Prisma Data Platform, this project should be available for you to access. It will also be synced with any schema changes pushed to the `master` branch on GitHub.
```suggestion
Alright! With all the configuration out of the way, click on the **Data Browser** buton on the left-hand side to get a hosted version of the Prisma Studio. The next time you log in to the Prisma Data Platform, this project should be available for you to access. It will also be synced with any schema changes pushed to the `master` branch on GitHub.
```